plugins/webp-uploads/image-edit.php

```diff
@@ -153,15 +153,19 @@ static function ( $metadata, $post_meta_id ) use ( $post_id, $file_path, $mime_t
 			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
 			$target = isset( $_REQUEST['target'] ) ? sanitize_key( $_REQUEST['target'] ) : 'all';
 
-			foreach ( $old_metadata['sizes'] as $size_name => $size_details ) {
-				// If the target is 'nothumb', skip generating the 'thumbnail' size.
-				if ( webp_uploads_image_edit_thumbnails_separately() && 'nothumb' === $target && 'thumbnail' === $size_name ) {
-					continue;
-				}
+			if ( isset( $old_metadata['sizes'] ) ) {
```

Comment:
The inclusion of this `if` statement wrapper is the only change to this block of code.

Reply:
turning off whitespace confirms this